Sources/Bootstrap/Bootstrap.swift

```diff
@@ -29,3 +33,25 @@ extension Array {
         return indices.contains(index) ? self[index] : nil
     }
 }
+
+/// Bootstrap Color Definitions
+///
+enum ColorKeys: String {
+    /// Bootstrap Primary Color
+    case primary    = "primary"
```

Comment:
Swift will automatically provide the raw values for these cases so they can be removed here.